sumatra/recordstore/django_store/__init__.py

```diff
@@ -279,8 +279,8 @@ def clear(self):
         if not db_config.configured:
             db_config.configure()
         #management.call_command('sqlclear', 'django_store', database=self._db_label)  # this produces coloured output, need no_color option from Django 1.7
-        cmds = ["BEGIN;"] + ['DROP TABLE "django_store_{}";'.format(x)
-                             for x in ("record", "record_output_data", "record_input_data", "record_dependencies",
```

Comment:
The table `django_store_record_output_data` no longer exists, because there is no longer a many-to-many relationship between output data files and records (since I merged the changes from @Felix11H 's GSoC project) . Previously, if running two computations produced two files with identical paths and identical content (but differing creation times) Sumatra treated these as a single data item. This made it very difficult to represent workflows/pipelines, so  Felix and I decided it was best to treat these as separate items. The information previously in `django_store_record_output_data` is now stored in `django_store_datakey`.

Input data items still has a many-to-many relationship, as you can use the same input file for many computations.